src/tests/ftest/cart/utest/utest_swim.c

```diff
@@ -56,7 +56,7 @@ init_tests(void **state)
 	unsigned int seed;
 
 	/* Seed the random number generator once per test run */
-	seed = time(NULL);
+	seed = (unsigned int)(time(NULL) & 0xFFFFFFFFUL);
```

Comment:
Should this be 0x0FFFFFFFFULL ?   What stops 0xffffffUL from getting sign extended?

Reply:
The number will be converted to unsigned int anyway. I think 0xFFFFFFFFUL and 0x0FFFFFFFFULL would behave same here. I can double check.

Reply:
It is confirmed that using 0xFFFFFFFFUL and 0x0FFFFFFFFULL behave exactly same since we only need the lower 32 bits.